BlueDB/src/main/java/org/bluedb/api/encryption/EncryptionConfig.java

```diff
@@ -0,0 +1,9 @@
+package org.bluedb.api.encryption;
+
+public interface EncryptionConfig {
+
+	String getCurrentEncryptionKeyVersion();
+	byte[] decrypt(String encryptionKeyVersion, byte[] bytesToDecrypt);
+	byte[] encrypt(String encryptionKeyVersion, byte[] bytesToEncrypt);
```

Comment:
My notes say `encryptionKeyVersion`, but I just realized that name is wrong. It is not the version of an encryption key. It should be `encryptionVersionKey`.

Reply:
Great catch, I'll update this!